server/main.js

```diff
@@ -56,11 +56,18 @@ const public = path.join(__dirname, "..", "public");
 app.use(express.static(public));
 
 io.on("connection", function(socket) {
-  let x = 0;
-  let y = 0;
+  let x,y;
+
   getSpawn(map, x, y);
```

Comment:
can we take this function out? What does it do?

Reply:
This function was used to find an empty spot on the map grid, then move it into the x/y variables. If it's not used, we can remove it.

Reply:
oh i see

Reply:
removing...